explainers/periodicsync-explainer.md

```diff
@@ -0,0 +1,178 @@
+task is run. This risk is also present with any tasks run in a service worker, but Periodic Sync
+allows persistent tracking. Note that this risk is also present with push messages,
+but they require explicit user permission.
+
+To mitigate these privacy concerns, the user agent can limit the number of times periodic tasks are
+run for a site if the user isn't currently visiting the site.
+
+An additional `periodic-background-sync` permission will be exposed through the Permissions API to
+allow developers to query whether the API can be used. The user or the user agent on behalf of the
+user must set this permission to 'granted' before the website can use this capability.
+
+# Design decisions
+
+## Separate interface or extending one-shot Background Sync?
+We think that the use cases for periodic background sync are sufficiently different from the use
+cases for one-shot Background Sync to justify having a separate event in the developer's service
+worker: synchronizing data in response to a user action versus opportunistically refreshing
+content. In addition, there are different timing guarantees: one-shot Background Sync has to run as
+soon as possible, where the browser has the final timing decision for periodic sync, as long as it
+honors the requested `minInterval` between two consecutive events.
+
+The interfaces mimic regular Background Sync, but substitute `navigator.sync` with
+`navigator.periodicSync` for symmetry with the event that will be fired in the service worker.
+
+## Power and Network conditions
+The original explainer allowed the website to specify power and network conditions under which the
+periodic task should be run. This proposal leaves those decisions to the browser implementation,
+because we believe the browser is better positioned to make those choices for the user.
+This also makes for a simpler interface for developers.
+
+When the device is in either battery or data saving mode, the `periodicsync` event SHOULD not be
+fired. This would also cover some situations where the browser incorrectly detects whether the
+device is connected to an unmetered connection.
+
+## Timing of periodic sync tasks
+The API allows the developer to specify a `minInterval` for each periodic sync task. The browser is
+expected to guarantee a minimum of this time interval between two periodic sync events. However, the
+interval between any two periodic sync events can be greater than the requested `minInterval`, and
+is up to the browser implemetation. Also, the interval between any two periodic sync events for a
+registration doesn't have to be the same.
+
+# References and acknowledgements
+
+  * [Background Sync](https://wicg.github.io/BackgroundSync/spec/)
+  * [Push API](https://w3c.github.io/push-api/)
+
+Many thanks to the authors of the original explainer for starting the project, and to @mugdhalakhani
+for picking periodic sync up again. Thanks to @jakearchibald and @rayankans for their ideas, input
+and discussion.
```

Comment:
Please update the acknowledgements, you should be the primary author, but keep mentioning the others.

Reply:
done